conf/node_deployment.ini

```diff
@@ -12,13 +12,15 @@ p2p_ip=127.0.0.1
 ; Please see https://tecadmin.net/check-ip-address-ubuntu-18-04-desktop/
 ; for more instructions.
 rpc_ip=127.0.0.1
+channel_ip=0.0.0.0

```

Comment:
maybe 127.0.0.1 will be better

Reply:
The default setting is 0.0.0.0 to support cross-machine operation